### PR TITLE
Support the Ring programming language

### PR DESCRIPTION
**Pull request by MahmoudFayed**
Hello

The package is updated to support the Ring programming language (http://ring-lang.net)

Greetings,
Mahmoud

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -165,6 +165,7 @@
             "applescript": "osascript",
             "clojure": "lein exec",
             "haxe": "haxe --cwd $dirWithoutTrailingSlash --run $fileNameWithoutExt",
+            "ring": "cd $dir && ring $fileName",
```
Comment (formulahendry) [inline package.json:168]: No need.

File: package.json [hunk 2]
```diff
@@ -219,7 +220,8 @@
             ".v": "v run",
             ".vsh": "v run",
             ".sass": "sass --style expanded",
-            ".cu": "cd $dir && nvcc $fileName -o $fileNameWithoutExt && $dir$fileNameWithoutExt"
+            ".cu": "cd $dir && nvcc $fileName -o $fileNameWithoutExt && $dir$fileNameWithoutExt",
+            ".ring": "cd $dir && ring $fileName"
           },
```
Comment (formulahendry) [inline package.json:225]: ".ring": "ring"

File: package.json (hunk 1 above)
Comment (MahmoudFayed) [reply package.json]: Removed

File: package.json (hunk 2 above)
Comment (MahmoudFayed) [reply package.json:224]: Updated

**Comment by MahmoudFayed** [comment]
Very nice, Keep up the GREAT WORK

**Pull request merged**